pycrefine.py

```
@@ -808,13 +792,333 @@ def _prescan_ternaries(self) -> None:
+
+            if not valid:
+                # Keep the scan moving if we can't merge it here.
+                i = group[0][0] + 1
+                continue
+
+            # -----------------------------------------------------------
+            # Build the combined condition string.
+            # -----------------------------------------------------------
+            parts: list = []  # list of (cond_str, is_or, jump_target)
+
+            for k, (jump_idx, jinstr, expr_instrs, _) in enumerate(group):
```

Ruff (0.15.7)

[warning] Loop control variable `k` not used within loop body

(B007)